### PR TITLE
{lang}[iccifort/2016.3.210-GCC-5.4.0-2.26] Python 2.7.12 (bare)

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/671db80a403ffbe385f01734263e3d27 for a full test report.

**Comment by boegel** [comment]
@vanzod Did you intend to update #4695 to include the bare Python rather than opening a new PR?

**Comment by vanzod** [comment]
@boegel No, this is not going to help with #4695 since for that PR a `GCCcore` version of `Python-2.7.12` is needed and I am not thinking about pursuing that route (at least not yet).

File: easybuild/easyconfigs/p/Python/Python-2.7.12-iccifort-2016.3.210-GCC-5.4.0-2.26-bare.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ('SQLite', '3.13.0'),
+    ('Tk', '8.6.5'),  # this requires a full X11 stack
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.0.2h'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+modluafooter = 'family("python")'
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.12-iccifort-2016.3.210-GCC-5.4.0-2.26-bare.eb]: site-specific customisation?

File: easybuild/easyconfigs/p/Python/Python-2.7.12-iccifort-2016.3.210-GCC-5.4.0-2.26-bare.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name = 'Python'
+version = '2.7.12'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'iccifort', 'version': '2016.3.210-GCC-5.4.0-2.26'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.12-iccifort-2016.3.210-GCC-5.4.0-2.26-bare.eb]: `optarch` and `opt` are enabled by default, no need to specify these (we need to do a cleanup sweep for those sometime...)

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2140.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/22a563a008403a45f0098d6a482f3cac for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2583.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0033805fc928d6af897c51449ce0c982 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**